src/request.ts

```diff
@@ -58,14 +58,13 @@ export async function request({
     url.toString(),
     {
       headers: {
-        ...(isPost
-          ? { "Content-Type": "application/json; charset=utf-8" }
-          : undefined),
+        ...
+        { "Content-Type": "application/json; charset=utf-8" },
```

Comment:
is it ok to always set the content type even if we don't post a body? I guess it doesn't matter right?

Reply:
I can't remember why I added it :thinking: 

According to https://stackoverflow.com/questions/5661596/do-i-need-a-content-type-header-for-http-get-requests it should only be added to POST / PUT requests. 

I'll try to revert.